src/cloud/docker/docker-containers-service.md

```diff
@@ -75,6 +75,19 @@ See [Important Elasticsearch configuration][] in the Elasticsearch documentation
 {:.bs-callout-info}
 If your Cloud project uses Magento version 2.3.5 or earlier with MySQL search, add the `--no-es` option to skip the Elasticsearch container configuration when you generate the Docker Compose configuration file: `ece-docker build:compose --no-es`.
 
+### Elasticsearch plugins
+
+Plugins `analysis-icu` and `analysis-phonetic` are installed by default and can not be skipped.
+Elasticsearch plugins configured in `.magento/services.yaml` will be automatically installed in elasticsearch container starting from version 6.5.
+You can change list of plugins in `docker-compose.yaml` file:
```

Comment:
```suggestion
The `analysis-icu` and `analysis-phonetic` plugins are installed by default and can not be skipped.
If you use Elasticsearch 6.5 and later, the Elasticsearch plugins configured in `.magento/services.yaml` are installed automatically.

You can change the list of plugins in `docker-compose.yaml` file:
```

Reply:
The plugin list will be automatically generated from plugins mentioned in `.magento/services.yaml` but it's possible to edit the generated `docker-compose.yaml` file to change plugin list.
So, the command will generate the next code in `docker-compose.yaml`
```
services:
    elasticsearch:
        environment:
          - 'ES_PLUGINS=analysis-stempel analysis-nori'
```
And before running `docker-compose up -d` it's possible to edit this environment variable manually for adding/removing required plugins.